static/sass/_pattern_footer.scss

```diff
@@ -1,44 +1,15 @@
 @mixin canonical-p-footer {
-  .p-footer {
-    @extend %vf-strip;
+  .row:first-child{
```

Comment:
This is going to affect every `row` on the site. What is it for?

Reply:
Only for the footer, first 2 lines weren't meant to be deleted